lib/exq/manager/server.ex

```diff
@@ -109,6 +109,21 @@ defmodule Exq.Manager.Server do
     Enqueuer.enqueue_in(state.enqueuer, from, queue, offset, worker, args)
     {:noreply, state, 10}
   end
+  
+  def handle_call({:subscribe, queue}, from, state) do
+    updated_state = add_queue(state, queue)
+    {:reply, :ok, updated_state}
```

Comment:
I know it's not documented, but these need to return a timeout after which to run again, something like:

```{:reply, :ok, updated_state, 0}```